speakers.js

```diff
@@ -0,0 +1,97 @@
+const speakers = [
+  {
+    picture: 'images/speakers/hamilton.png',
+    name: 'Lewis Hamilton',
+    description: '7-Times World Champion',
+    text: 'This amazing driver won his first World Championship with McLaren in 2008',
+  },
+  {
+    picture: 'images/speakers/mika.png',
+    name: 'Mika Hakkinen',
+    description: '2-Times World Champion',
+    text: 'Mika won his two World Championship Titles with McLaren in 1998 and 1999',
+  },
+  {
+    picture: 'images/speakers/norris.png',
+    name: 'Lando Norris',
+    description: 'McLaren F1 Driver',
+    text: 'Racing for McLaren since 2019, he is our young talent',
+  },
+  {
+    picture: 'images/speakers/brown.png',
+    name: 'Zak Brown',
+    description: 'McLaren Racing Team Principal',
+    text: 'Chief Director of McLaren Motorsport division',
+  },
+  {
+    picture: 'images/speakers/driver.png',
+    name: 'Mike Turner',
+    description: 'McLaren E-Sports Driver',
+    text: 'Racing for McLaren E-Sports since 2019, he is our young talent',
+  },
+  {
+    picture: 'images/speakers/driver2.png',
+    name: 'Matt Helders',
+    description: 'McLaren E-Sports Driver',
+    text: 'Racing for McLaren E-Sports since 2019, he is our young talent',
+  },
+];
+
+document.getElementById('spk-img0').src = speakers[0].picture;
+document.getElementById('spk-title0').textContent = speakers[0].name;
+document.getElementById('spk-subTitle0').textContent = speakers[0].description;
+document.getElementById('spk-text0').textContent = speakers[0].text;
+
+document.getElementById('spk-img1').src = speakers[1].picture;
+document.getElementById('spk-title1').textContent = speakers[1].name;
+document.getElementById('spk-subTitle1').textContent = speakers[1].description;
+document.getElementById('spk-text1').textContent = speakers[1].text;
+
+document.getElementById('spk-img2').src = speakers[2].picture;
+document.getElementById('spk-title2').textContent = speakers[2].name;
+document.getElementById('spk-subTitle2').textContent = speakers[2].description;
+document.getElementById('spk-text2').textContent = speakers[2].text;
+
+document.getElementById('spk-img3').src = speakers[3].picture;
+document.getElementById('spk-title3').textContent = speakers[3].name;
+document.getElementById('spk-subTitle3').textContent = speakers[3].description;
+document.getElementById('spk-text3').textContent = speakers[3].text;
+
+document.getElementById('spk-img4').src = speakers[4].picture;
+document.getElementById('spk-title4').textContent = speakers[4].name;
+document.getElementById('spk-subTitle4').textContent = speakers[4].description;
+document.getElementById('spk-text4').textContent = speakers[4].text;
+
+document.getElementById('spk-img5').src = speakers[5].picture;
+document.getElementById('spk-title5').textContent = speakers[5].name;
+document.getElementById('spk-subTitle5').textContent = speakers[5].description;
+document.getElementById('spk-text5').textContent = speakers[5].text;
```

Comment:
- [ ] Please use a loop to create the speaker's section dynamically. You should create every element related to this section in the loop in your `.js` file.